phpBB/phpbb/template/twig/extension.php

```diff
@@ -93,6 +93,7 @@ public function getFunctions()
 			new \Twig\TwigFunction('lang', array($this, 'lang')),
 			new \Twig\TwigFunction('lang_defined', array($this, 'lang_defined')),
 			new \Twig\TwigFunction('lang_js', [$this, 'lang_js']),
+			new \Twig\TwigFunction('file_exists', [$this, 'on_disk']),
```

Comment:
If I don't remember wrong, you can just do
```php
new \Twig\TwigFunction('file_exists', 'file_exists'),
```

Reply:
Yeah but I want to use `if file_exists()` not `if on_disk()` in the markup.

Reply:
Then do
```php
new \Twig\TwigFunction('file_exists', 'file_exists'),
```
just like i did in with get_class

Reply:
^^ see above, yup